packages/exchange-analytics/src/models/funding-rate.ts

```diff
@@ -0,0 +1,65 @@
+import { AdapterError } from '@chainlink/ea-bootstrap'
+import { datetime } from '@linkpool/shared'
+
+import { BinanceFundingRatesGetPayload } from '../api/binance/types'
+import { BybitFundingRatesGetPayload } from '../api/bybit/types'
+import { Provider } from '../api/constants'
+import { FtxFundingRatesGetPayload } from '../api/ftx/types'
+
+export interface IFundingRate {
+  nextFunding: {
+    rate: number
+    timestamp: number
+  }
+}
+
+const transformer = {
+  [Provider.BINANCE]: (payload: BinanceFundingRatesGetPayload): IFundingRate => {
+    return {
+      nextFunding: {
+        rate:
+          (Number(payload.markPrice) - Number(payload.indexPrice)) /
+          Number(payload.indexPrice) /
+          24,
```

Comment:
Could you create meaningful constants for these magic numbers please? e.g. `export const HOURS_IN_A_DAY = 24`